src/ONE.Models.CSharp/ONE.Models.CSharp.csproj

```diff
@@ -18,8 +18,8 @@
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
     <PackageRequireLicenseAcceptance>True</PackageRequireLicenseAcceptance>
     <PackageReleaseNotes>
-      Version 4.6.0
-      - Added Analyte, TestAnalyteGroup, TestAnalyteGroupItem
+      Version 5.0.0
```

Comment:
Is anyone using ScheduleOccurence?  If not, I'd vote maybe make it a bugfix increment.

Reply:
The version will be 4.6.1 for a bug fix. We will have to make 4.6.0 not visible, though, so nobody can download it.